docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,576 +1,641 @@
+<!-- TO BE DONE:
+(the full list of a `View`'s properties can be found in the [View section](./view.md))
+-->
+
+A `VisualView` stores the handles to visuals and maps of their properties. You can find the total number of visuals that are added to the `VisualView` using the `NumberOfVisuals`. The following are some of the useful methods for working with visuals:
+
+-  To add a visual to the `VisualView`, use the `AddVisual(string, VisualMap)`. The method expects the following two parameters:
+   - `string` - specifies the name of the visual, by which it is identified within a given `VisualView`,
+   - `VisualMap` - specifies the visual, that will be added to the `VisualView`. If the visual with a given name already exists in the `VisualView`, it will be updated instead of getting added.
+
+-  To remove a visual from a `VisualView`, use the `RemoveVisual(string)`, its only parameter specifies the name of the visual.
+
+-  To remove all the visuals from a `VisualView`, use `RemoveAll()`.
+    
+The following example shows how to create a `VisualView`, set some of the properties, and use all of the mentioned methods:
+   ```csharp
+   // Create VisualView
+   VisualView _visualView = new VisualView();
+   // Set the background color
+   _visualView.BackgroundColor = Color.Blue;
+   // Set the absolute size of the VisualView
+   _visualView.Size2D = new Size2D(300, 300);
+   // The parent's reference point
+   _visualView.ParentOrigin = ParentOrigin.TopRight;
+   // Must be 'true' in order to use the PivotPoint
+   _visualView.PositionUsesPivotPoint = true;
+   // The VisualView reference point
+   _visualView.PivotPoint = PivotPoint.TopRight;
+   // Shifting between ParentOrigin and PivotPoint in absolute units
+   _visualView.Position2D = new Vector2(-10, 10);
+
+   // Create visual 1
+   ColorVisual _colorVisual_1 = new ColorVisual();
+   _colorVisual_1.MixColor = Color.Green;
+   _colorVisual_1.RelativeSize = new RelativeVector2(0.7f, 0.6f);
+   _colorVisual_1.Origin = Visual.AlignType.BottomEnd;
+   _colorVisual_1.AnchorPoint = Visual.AlignType.BottomEnd;
+
+   // Add visual 1 to VisualView
+   _visualView.AddVisual("colorVisual_1", _colorVisual_1);
+
+   // Create visual 2
+   ColorVisual _colorVisual_2 = new ColorVisual();
+   _colorVisual_2.MixColor = Color.Magenta;
+   _colorVisual_2.RelativeSize = new RelativeVector2(0.2f, 0.3f);
+   _colorVisual_2.Origin = Visual.AlignType.Center;
+   _colorVisual_2.AnchorPoint = Visual.AlignType.TopBegin;
+
+   // Add visual 2 to VisualView
+   _visualView.AddVisual("colorVisual_2", _colorVisual_2);
+
+   // Remove visual 1 - identified by its name
+   _visualView.RemoveVisual("colorVisual_1");
+
+   // Remove all visuals if any left
+   if (_visualView.NumberOfVisuals > 0)
+       _visualView.RemoveAll();
+
+   ```
+
+The result of the preceding code is as follows:
```

Comment:
Thank you for this comment.
I made slight changes, because we don't implement these classes in the preceding examples. 

I change to (and similarly in the others):
Line 193:  The final output after implementing the preceding code, and setting the `_visualView` background color to gray is as follows: